README.md

```diff
@@ -266,6 +266,12 @@ application, you should __always__ benchmark if performance matters to you.
 Please visit [Fastify help](https://github.com/fastify/help) to view prior
 support issues and to ask new support questions.
 
+Version 3 of Fastify and lower are EOL and will not receive any security or bug fixes.
+
+Our partner, HeroDevs, provides commercial security fixes for all unsupported versions
```

Comment:
https://fastify.dev/docs/latest/Guides/Style-Guide/

"You"/"our" is fine for any informal documentation in `docs/Guides` but not for anything in `docs/Reference`.

Reply:
Corrected where needed.